packages/table/src/cell/formats/jsonFormat.tsx

```diff
@@ -38,15 +38,28 @@ export class JSONFormat extends React.Component<IJSONFormatProps, {}> {
 
     public render() {
         const { children, omitQuotesOnStrings, stringify } = this.props;
+
+        const isNully = children === undefined || children === null;
```

Comment:
nit: the conventional style in Blueprint seems to be to do a double-equal comparison for `null`/`undefined` checks:

```
const isNully = children == null;
```